calico_containers/calico_ctl/container.py

```diff
@@ -133,7 +134,7 @@ def container(arguments):
         elif arguments.get("endpoint"):
             orchestrator_id, workload_id = \
                                   lookup_workload(arguments.get("<CONTAINER>"))
-            endpoint.endpoint_show(hostname,
+            endpoint.endpoint_show(get_hostname(),
```

Comment:
Not if you're using it just the once.  However, you are calling this function several times in a row - which is generally less efficient than storing the result in a local and using that same result multiple times.